autotest/gdrivers/vrtovr.py

```diff
@@ -68,6 +68,12 @@ def test_vrtovr_1():
     fl = ds.GetFileList()
     assert fl == ["data/byte.tif", "data/int16.tif"], "did not get expected file list"
 
+    ds = gdal.Open("/vsigzip/data/rmf/overviews.rsw.gz")
+    assert ds.GetRasterBand(1).GetOverviewCount() == 3
+
+    ds = gdal.Open("vrt:///vsigzip/data/rmf/overviews.rsw.gz?ovr=1")
+    assert ds.GetRasterBand(1).GetOverviewCount() == 0
+
```

Comment:
- I would see this checks more in the existing test_vrt_protocol() of autotest/gcore/vrt_read.py
- The RMF driver is not built in the minimum configurations. You'd better use a GeoTIFF source. I'd suggest doing "cp autotest/gcore/data/byte.tif autotest/gcore/data/byte_with_ovr.tif; gdaladdo autotest/gcore/data/byte_with_ovr.tif 2 4", and testing with that file
- Rather than testing GetOveriewCout(), it would be more convincing to test ds.RasterXSize